### PR TITLE
Add test delete command

### LINKED ISSUE #49: Add more tests for delete command
Improve coverage by adding more tests for delete command and delete command parser

### PR DESCRIPTION
**Pull request by ivorcmx**
Improve coverage by adding more tests for DeleteTaskCommand and DeleteTaskCommandParser

### CODE REVIEW

**Comment by ivorcmx** [comment]
Fixes Issue #49

File: src/test/java/seedu/address/logic/parser/DeleteTaskCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @Test
+    public void parse_validArgs_returnsDeleteCommand() {
+        assertParseSuccess(parser, "1", new DeleteTaskCommand(1));
+    }
+
+    @Test
+    public void parse_invalidArgs_throwsParseException() {
+        assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
+    }
+
```
Comment (putaojuice) [inline src/test/java/seedu/address/logic/parser/DeleteTaskCommandParserTest.java:34]: Could add in more tests like using a mix of symbols as the arguments.

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 2]
```diff
@@ -64,9 +65,11 @@ private TypicalPersons() {} // prevents instantiation
      */
     public static AddressBook getTypicalAddressBook() {
         AddressBook ab = new AddressBook();
+        TaskList taskList = new TaskList();
```
Comment (putaojuice) [inline src/test/java/seedu/address/testutil/TypicalPersons.java]: what is the purpose of creating the TaskList here?

**Review by putaojuice — COMMENTED** [review]
Test coverages are fine, not sure why you need to include `TaskList` into `TypicalPersons`, i don't see how they should be related in terms of OOP design.

File: src/test/java/seedu/address/logic/parser/DeleteTaskCommandParserTest.java (hunk 1 above)
Comment (ivorcmx) [reply src/test/java/seedu/address/logic/parser/DeleteTaskCommandParserTest.java:34]: Added as requested. An additional test will be run on invalid symbols

File: src/test/java/seedu/address/testutil/TypicalPersons.java (hunk 2 above)
Comment (ivorcmx) [reply src/test/java/seedu/address/testutil/TypicalPersons.java]: Good catch. Apologies, unnecessary line of code has been removed.

**Review by putaojuice — APPROVED** [review]
LGTM

**Pull request merged**